src/cpuinfo_x86.c

```diff
@@ -1043,6 +1043,8 @@ static void ParseLeaf4(const int max_cpuid_leaf, CacheInfo* info) {
 static void ParseCpuId(const uint32_t max_cpuid_leaf, X86Info* info) {
   const Leaf leaf_1 = SafeCpuId(max_cpuid_leaf, 1);
   const Leaf leaf_7 = SafeCpuId(max_cpuid_leaf, 7);
+  const Leaf leaf_80000000 = CpuId(2147483648);
```

Comment:
This code should be conditionally executed since it's only valid for AMD CPUs.
To be honest I'd rather have a separate `ParseExtraAMDCpuId` dealing with AMD only features.

Reply:
maybe here for the call https://github.com/google/cpu_features/blob/eb168a2da2e14a85834b98c3961eee909dc618f9/src/cpuinfo_x86.c#L1135-L1137

Reply:
@gadoofou87  on my way to patch this PR...